<launch>
  <arg name="default_map_name" default="map"/>
  <arg name="add_waypoint_topic" default="/initialpose"/>
  <arg name="connect_movebase" default="true"/>
  <arg name="enable_saving" default="true"/>
  <arg name="waypoint_server_name" default="mission_path_srv"/>
  <arg name="web_ui_url" default="http://www.ross-robotics.co.uk"/>
  <arg name="web_ui_url_tab" default="https://www.bbc.co.uk"/>
  <arg name="move_base_namespace" default="/move_base"/>
  <arg name="build_bt_srv" default="/build_bt"/>

  <node pkg="rr_viz" type="rr_viz.py" name="rr_viz" output="screen" required="false">
    <param name="default_map_name" value="$(arg default_map_name)"/>
    <param name="add_waypoint_topic" value="$(arg add_waypoint_topic)"/>
    <param name="connect_movebase" value="$(arg connect_movebase)"/>
    <param name="enable_saving" value="$(arg enable_saving)"/>
    <param name="waypoint_server_name" value="$(arg waypoint_server_name)"/>
    <param name="web_ui_url_tab" value="$(arg web_ui_url_tab)"/>
    <param name="web_ui_url" value="$(arg web_ui_url)"/>
    <param name="move_base_namespace" value="$(arg move_base_namespace)"/>
    <param name="build_bt_srv" value="$(arg build_bt_srv)"/>

  </node>
</launch>
